src/py/flwr/common/record/parametersrecord.py

```diff
@@ -83,33 +83,82 @@ def _check_value(value: Array) -> None:
 
 
 class ParametersRecord(TypedDict[str, Array]):
-    """Parameters record.
+    r"""Parameters record.
```

Comment:
May I know why here's an `r`?

Reply:
is because there are a few `\` in the examples (when i show the print of the binary data).